consultation_analyser/consultations/download_consultation.py

```diff
@@ -7,6 +8,8 @@
     ConsultationWithResponsesAndThemes,
 )
 
+logger = logging.getLogger("download_json")
```

Comment:
I don't think this will work unless you define the logger in the settings file (should be one line to copy paste!)

Reply:
Thanks!!